clientd3d/move.c

```diff
@@ -773,21 +774,30 @@ void MoveUpdatePosition(void)
 {
    int x, y;
    BYTE speed;
+   DWORD current_time;
+   time_t rawtime;
+   struct tm * timeinfo;
+   char buffer[9]; // Buffer to hold HH:MM:SS

```

Comment:
Declare variables at point of first use instead of at the beginning of the function, where possible.